### ISSUE TITLE
right config directory

### ISSUE THREAD

**Issue by pilot1981** [tjntomas/HADashboard-widgets#6]
Hi,
I have HASSIO version 0.91, but I don't find this directory: /conf/custom_widgets folder

It's the same create this one? 

/config/custom_widgets

I have to declare it anywhere?

regards

**Comment by tjntomas** [comment]
Yes, it is the right directory to put the folder and files in. I will add a note that the folder is /config/custom_widgets for those using hassio. And no, you dont have to declare it anywhere.

**Comment by pilot1981** [comment]
thanks for quick reply!

so the result in my HASSIO could be this:

- /config/custom_widgets/basevacuum folder
- /config/custom_widgets/vacuum.yaml file
- /config/custom_css/img folder
- /config/custom_css/img folder

?

**Comment by pilot1981** [comment]
or you suggested:

- /config/appdaemon/conf/custom_widgets/basevacuum folder
- /config/appdaemon/conf/custom_widgets/vacuum.yaml file
- /config/appdaemon/conf/custom_css/img folder
- /config/appdaemon/conf/custom_css/img folder

?

**Comment by pilot1981** [comment]
Hi,
I found and tested successful the right directory in HASSIO v 0.91.0:

/config/appdaemon/custom_widgets/basevacuum folder
/config/appdaemon/custom_widgets/vacuum.yaml file
/config/appdaemon/custom_css/img folder

in this way no error are found in log file, but when I go in the dashboard I receive this error:

Unable to find widget type 'vacuum'

what else it's wrong?

**Comment by tjntomas** [comment]
The appdaemon folder in hassio appears to be /config/appdaemon so your last post is correct.

**Comment by tjntomas** [comment]
Did you restart appdaemon?

**Comment by pilot1981** [comment]
yes, I understood I have to write a more complete dashboard file as this one in Roomba.dash file:

title: Roomba
widget_dimensions: [120, 120]
widget_margins: [5, 5]
columns: 8

label:
    widget_type: label
    text: My vacuum cleaner

vacuum:
  widget_type: vacuum
  entity: switch.roomba
  title: Roomba 521

layout:
- vacuum(3x2)

vacuum_widget_style: $background_style
vacuum_title_style: $style_title

Now I have a problem to find entity object, but this isn't problem of your code..I let you know when all work fine to finish integration...

**Comment by pilot1981** [comment]
sorry a warning in my logs:

2019-04-06 00:02:33.949349 WARNING Variable definition not found in CSS Skin variables: $vacuum_title_style
2019-04-06 00:02:33.950146 WARNING Variable definition not found in CSS Skin variables: $vacuum_widget_style

you know how to solve?

**Comment by tjntomas** [comment]
vacuum_widget_style: $background_style
vacuum_title_style: $style_title

These entries need to be in a custom skin in the variables.yaml file. IF you don't have a custom skin, make a new folder in custom_css and name it default. Then copy varaibles.yaml and dashboard.css from [here](https://github.com/home-assistant/appdaemon/tree/dev/appdaemon/assets/css/default). THe nedit the varialbes.yaml file and add the vacuum_widget_style and vacuum_title_style lines to the file.
